authors/apps/authentication/views.py

```diff
@@ -72,3 +81,61 @@ def update(self, request, *args, **kwargs):
         serializer.save()
 
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ResetPasswordEmail(CreateAPIView):
+    """Password reset emailing with uid and reset token"""
+    permission_classes = (AllowAny,)
+    serializer_class = ResetEmailSerializer
+
+    @classmethod
+    def post(cls, request, *args, **kwargs):
+        """Returns message for reset password email"""
+        data = request.data
+        serializer = cls.serializer_class(data=data)
+        serializer.is_valid(raise_exception=True)
+        user = User.objects.get(email=data['email'])
+        body = json.dumps({
```

Comment:
@ManuelDominic , the PT board story says you should send a link to the email. Could you please ensure that only a link to reset URL is sent.